.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: cpp
+
+sudo: false
+
+os:
+  - linux
+
+compiler:
+  - g++-4.9
+
+script:
+    - make CXX=g++-4.9 test
+
+addons:
+  apt:
+    sources:
+    - deadsnakes
+    - ubuntu-toolchain-r-test
```

Comment:
Do we need the `deadsnakes` ppa?  I assume the ubuntu-toolchain-r-test one is for g++?